languages/java8/template/src/main/java/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,26 @@
+package algorithmia;
+import com.algorithmia.algorithmHandler.*;
+import com.algorithmia.*;
+import java.util.HashMap;
+
+public class Algorithm {
+    String Apply(String input, HashMap<String, String> context) throws Exception{
+            if(context.containsKey("local_file")){
+                return "hello ".concat(input) + " your model file is downloaded here ".concat(context.get("local_file"));
+            }
+        return "hello ".concat(input);
```

Comment:
Nit: indentation is inconsistent: is this a tabs vs spaces issue?

Reply:
It might be, I'll throw it into a linter.